scripts/install/requestrr.sh

```diff
@@ -0,0 +1,182 @@
+      "ClientId": "",
+      "StatusMessage": "!help",
+      "TvShowRoles": [],
+      "MovieRoles": [],
+      "MonitoredChannels": [],
+      "EnableRequestsThroughDirectMessages": false,
+      "AutomaticallyNotifyRequesters": true,
+      "NotificationMode": "PrivateMessages",
+      "NotificationChannels": [],
+      "AutomaticallyPurgeCommandMessages": false,
+      "DisplayHelpCommandInDMs": true
+    }
+  },
+  "DownloadClients": {
+    "Ombi": {
+      "Hostname": "",
+      "Port": 3579,
+      "ApiKey": "",
+      "ApiUsername": "",
+      "BaseUrl": "",
+      "UseSSL": false,
+      "Version": "3"
+    },
+    "Overseerr": {
+      "Hostname": "",
+      "Port": 5055,
+      "ApiKey": "",
+      "DefaultApiUserID": "",
+      "UseSSL": false,
+      "Version": "1"
+    },
+    "Radarr": {
+      "Hostname": "${r_address}",
+      "Port": ${r_port},
+      "ApiKey": "${r_key}",
+      "BaseUrl": "/${r_base}",
+      "MovieProfileId": "1",
+      "MovieRootFolder": "",
+      "MovieMinimumAvailability": "",
+      "MovieTags": [],
+      "AnimeProfileId": "1",
+      "AnimeRootFolder": "",
+      "AnimeMinimumAvailability": "",
+      "AnimeTags": [],
+      "SearchNewRequests": true,
+      "MonitorNewRequests": true,
+      "UseSSL": false,
+      "Version": "2"
+    },
+    "Sonarr": {
+      "Hostname": "${s_address}",
+      "Port": ${s_port},
+      "ApiKey": "${s_key}",
+      "BaseUrl": "/${s_base}",
+      "TvProfileId": "1",
+      "TvRootFolder": "",
+      "TvTags": [],
+      "TvLanguageId": "1",
+      "TvUseSeasonFolders": true,
+      "AnimeProfileId": "1",
+      "AnimeRootFolder": "",
+      "AnimeTags": [],
+      "AnimeLanguageId": "1",
+      "AnimeUseSeasonFolders": true,
+      "SearchNewRequests": true,
+      "MonitorNewRequests": true,
+      "UseSSL": false,
+      "Version": "3"
+    }
+  },
+  "BotClient": {
+    "Client": "",
+    "CommandPrefix": "!"
+  },
+  "Movies": {
+    "Client": "Disabled",
+    "Command": "movie"
+  },
+  "TvShows": {
+    "Client": "Disabled",
+    "Command": "tv",
+    "Restrictions": "None"
+  },
+  "Port": 4545,
+  "BaseUrl" : "/requestrr",
+  "Version": "1.12.0"
+}
+CFG
+
+echo_progress_done "Requestrr config applied."
+
+echo_progress_start "Installing Systemd service"
+cat > /etc/systemd/system/requestrr.service << EOF
+[Unit]
+Description=Requestrr Daemon
+After=syslog.target network.target
+
+[Service]
+User=requestrr
+Type=simple
+WorkingDirectory=/opt/requestrr/
+ExecStart=/opt/requestrr/Requestrr.WebApi
+TimeoutStopSec=20
+KillMode=process
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+if [[ -f /install/.nginx.lock ]]; then
+    echo_progress_start "Installing nginx configuration"
+    bash /usr/local/bin/swizzin/nginx/requestrr.sh
+    systemctl -q reload nginx
+    echo_progress_done "Nginx configured"
+else
+    echo_info "Requestrr will be available on port 4545. Secure your installation manually through the web interface."
```

Comment:
If the username and the salted password hash is in the config, does this still need to be done manually?

Reply:
this will be updated once I've committed it.